Index: src/main/java/pe/edu/tecsup/lab03/repositories/StudentController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/pe/edu/tecsup/lab03/repositories/StudentController.java b/src/main/java/pe/edu/tecsup/lab03/repositories/StudentController.java
new file mode 100644
--- /dev/null	(date 1725479259001)
+++ b/src/main/java/pe/edu/tecsup/lab03/repositories/StudentController.java	(date 1725479259001)
@@ -0,0 +1,4 @@
+package pe.edu.tecsup.lab03.repositories;
+
+public class StudentController {
+}
